examples/mautic-example-nginx-ssl/nginx.conf

```
@@ -1,8 +1,12 @@
 server {
-	listen 443 ssl;
-    ssl_certificate     /etc/ssl/private/mautic.crt;
-    ssl_certificate_key /etc/ssl/private/mautic.key;
-	location / {
+        listen 443 ssl default_server;
+        ssl_dhparam /etc/ssl/certs/dhparam-2048.pem;
+        ssl_certificate     /etc/letsencrypt/live/beta-mautic.thrivebrokers.com/cert.pem;
+        ssl_certificate_key /etc/letsencrypt/live/beta-mautic.thrivebrokers.com/privkey.pem;
+
+        #ssl_certificate     /etc/letsencrypt/archive/beta-mautic.thrivebrokers.com/cert1.pem;
+        #ssl_certificate_key /etc/letsencrypt/archive/beta-mautic.thrivebrokers.com/privkey1.pem;
```

As this is commented out, do we need to retain it?

```suggestion
        #ssl_certificate_key /etc/letsencrypt/archive/beta-mautic.example.com/privkey1.pem;
```